packages/utils/cli/package.json

```diff
@@ -11,9 +11,14 @@
     "spark-setup-themes": "./bin/spark-setup-themes.mjs"
   },
   "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:adevinta/spark.git",
+    "directory": "packages/utils/cli"
+  },
   "dependencies": {
     "@clack/prompts": "0.6.2",
-    "@spark-ui/theme-utils": "^2.9.0",
+    "@spark-ui/theme-utils": "2",
```

Comment:
shouldn't we use the "latest" keyword to ensure we always have the most up-to-date version?

Reply:
But, `latest` tag can contain breaking changes... so I think is better to keep the control for major versions (of our own packages), minors and bugfixes will be always autoupdated.